src/operator/extended/distinctUntilKeyChanged.ts

```diff
@@ -1,10 +1,10 @@
 import {distinctUntilChanged} from '../distinctUntilChanged';
 
-export function distinctUntilKeyChanged<T>(key: string, compare?: (x: any, y: any) => boolean, thisArg?: any) {
+export function distinctUntilKeyChanged<T>(key: string, compare?: (x: any, y: any) => boolean) {
   return distinctUntilChanged.call(this, function(x, y) {
```

Comment:
actually this also seems bit .. to be modified. is this function need to call `distinctUntilChanged` with `this` context?